doc/build-instructions/Build_Instructions_V11.md

```diff
@@ -457,10 +457,20 @@ You must install a number of software dependencies to create a suitable build en
 
 - [Xcode 9.4]( https://developer.apple.com/download/more/) (requires an Apple account to log in).
 - [macOS OpenJDK 10](https://adoptopenjdk.net/archive.html?variant=openjdk10&jvmVariant=hotspot), which is used as the boot JDK.
-- [nasm 2.13.03](https://formulae.brew.sh/formula/nasm), which can be installed by using [Homebrew](https://brew.sh/).
-- [bash 4.4.23](https://formulae.brew.sh/formula/bash), which can be installed by using [Homebrew](https://brew.sh/).
-- [gnu-tar 1.3](https://formulae.brew.sh/formula/gnu-tar), which can be installed by using [Homebrew](https://brew.sh/).
-- [Freemarker V2.3.8](https://sourceforge.net/projects/freemarker/files/freemarker/2.3.8/freemarker-2.3.8.tar.gz/download), which can be obtained and installed with the following commands:
+
+The following dependencies can be installed by using [Homebrew](https://brew.sh/):
+
+- [autoconf 2.6.9](https://formulae.brew.sh/formula/autoconf)
+- [bash 4.4.23](https://formulae.brew.sh/formula/bash)
```

Comment:
I don't think `bash 4` is required for JDK11. There is no harm in using `bash 4`. `bash 4` can be optional. if `bash 4` is used, then the following steps should be mentioned:

Bash version 4 is installed to `/usr/local/bin/bash`. To prevent problems during the build process, make Bash v4 your default shell by typing the following commands:

```
# Find the <CURRENT_SHELL> for <USERNAME>
dscl . -read <USERNAME> UserShell

# Change the shell to Bash version 4 for <USERNAME>
dscl . -change <USERNAME> UserShell <CURRENT_SHELL> /usr/local/bin/bash

# Verify that the shell has been changed
dscl . -read <USERNAME> UserShell
```